src/md/src/Analysis/AutoAnalysisOfLaunches.md

```diff
@@ -42,90 +40,249 @@ The following info is sent:
 * A launch name;
 * Unique ID;
 
-So that the more you analyze, the more data in the ElasticSearch, the more accurate results you will get in the end of the process. 
+Service Analyzer forms a query with following data to the  service ElasticSearch.:
+* LogMessage;
+* Launch ID;
+* UniqueID test item; 
+* Defect type;
+
+So each log with its defect type is saved in ElasticSearch as a separate document. All document creates Index. So the more test results you analyze , the more data in the ElasticSearch appears, the more accurate results you will get in the end of the process. 
+
+>Note: You need to send at least **N logs (where N is a value of parameter MinDocFreq)** to ElasticSearch for start to analyze test results automatically.
+>
+>If you do not sure how many documents(logs) are contained in the Index on that moment, you can check it. 
+>For that, perform following actions: 
+>* Uncommented Service ElasticSearch ports in docker-compose file or add them: 9200:9200; 
+>* Restart Service ElasticSearch with new docker-compose; 
+>* Send a request to ElasticSeaarch: 
+>    * how many documents in the Index: GET http://localhost:9200/_cat/indices?v  
+>    * Detailed information:  POST http://localhost:9200/{project_name}/_search
+
+Test items of a launch in Debug mode are not sent to service Analyzer. If you delete test item or move it to the Debug, the test item is removed from a base.
```

Comment:
please, use passive voice. "... if test item is deleted or moved to Debug mode, it's being deleted from the base" (what is 'base' btw?)